File: teste.c
#include <stdio.h>

int primo(int num)
{
	int i;
	if (num < 2)
		return 0;
	for (int i = 2; i < num; i++)
	{
		if (num % i == 0)
			return 0;
	}
	return 1;
}

int main()
{
	for (int i = 0; i < 100000; i++)
	{
		if (primo(i))
			printf("%d\n", i);
	}
	return 0;
}

File: teste.php
<?php
	function primo($num){
		if($num < 2)
			return false;
		for($i = 2; $i < $num; $i++){
			if($num % $i == 0)
				return false;
		}
		return true;
	}

	for($i = 0; $i < 100000; $i++){
		if(primo($i))
			echo $i."\n";
	}
?>